libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -2770,6 +2822,20 @@ add_fp_unittest(
     libc.src.__support.FPUtil.normal_float
 )
 
+add_fp_unittest(
+  scalbnf128_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    scalbnf128_test.cpp
+  HDRS
+    ScalbnTest.h
+  DEPENDS
+    libc.src.math.scalbnf128
+    libc.src.__support.FPUtil.fp_bits
+    libc.src.__support.FPUtil.normal_float
```

Comment:
Look like `FPUtil.normal_float` is not directly used in the test.  You can remove it from the dependency list.

Reply:
I removed the dependency from the other scalbn tests also.